### ISSUE TITLE
OMP warning at runtime when compiling with Intel MPI 2020.2

### ISSUE THREAD

**Issue by MatthieuSchaller** [ICRAR/VELOCIraptor-STF#22]
We are in the process of upgrading our compilation tool stack to Intel MPI 2020 (2020.2 to be specific). The code compiles without problems (`cmake ../ -DVR_USE_HYDRO=ON -DCMAKE_BUILD_TYPE=Release -DVR_MPI=off`) but at runtime, I get the following messages:

```
OMP: Info #274: omp_get_nested routine deprecated, please use omp_get_max_active_levels instead.
OMP: Info #274: omp_set_nested routine deprecated, please use omp_set_max_active_levels instead.
```

The code keeps running and seems to produce correct results nevertheless.

This happens in between 
```
First build tree ... 
```
and
```
0: finished building 64 domains and trees
```

**Comment by MatthieuSchaller** [comment]
Actually it also happens in later sections of the run.

**Comment by rtobar** [comment]
Reproduced locally too with clang 10 in Ubuntu 20.04. The OpenMP library clang links against implements 4.5, but this warning message is already there, so I can see it too.

**Comment by rtobar** [comment]
Potentially fixed by #25, waiting for confirmation.

**Comment by rtobar** [comment]
Fixed by #25.
